region/norfair/east/Norfair Reserve Tank Room.json

```diff
@@ -570,6 +576,17 @@
       "clearsObstacles": ["A"],
       "devNote": "Ice can move through the room more quickly but must spend time refreezing the dragons."
     },
+    {
+      "id": 32,
+      "link": [4, 1],
+      "name": "Return Carefully (Sovas Cleared)",
+      "requires": [
+        {"obstaclesCleared": ["B"]},
+        {"obstaclesNotCleared": ["A"]},
+        {"heatFrames": 570}
```

Comment:
copy-pasted frame count from the "kill" strat.. not sure what the right number is

Reply:
```suggestion
        {"heatFrames": 480}
```